tests/env/typescript-node/src/index.ts

```diff
@@ -15,10 +15,13 @@ const config = {
   apiKey: 'masterKey',
 }
 
-interface Movie {
+export interface Movie {
   id: number
   title: string
   genre?: string
+  comment?: string
+  isNull?: null
+  isTrue?: true
 }
```

Comment:
```suggestion
interface Movie {
  id: number
  title: string
  genre?: string
}
```

Reply:
https://github.com/meilisearch/meilisearch-js/pull/1416/files#file-tests-typed_search-test-ts-L285

like the error we already has here

Reply:
I see, indeed it should stay like this in this file. Could you duplicate this type in `search.test.ts` and `type_search.test.ts` and not import it from here?

Reply:
if you don't want to import this type we can just remove it from dataset

Reply:
This env is used during tests to ensure I can build in typescript-node. Seems a bit weird but I really much want to be sure that the package runs in most environments. 

Lets leave it like this for the moment :)